### DISCUSSION TITLE
Trigger relay every other day.

### DISCUSSION THREAD

**Discussion by MakoShark2** [theyosh/TerrariumPI#962] [Q&A] [answered]
Daily scheduling works fine, but Is it possible to schedule a relay every other day?

For example, I would like the mist pump to actuate 30 seconds at 20:00 on every second day.

Thanks.

**Comment by theyosh** [comment, marked as answer]
Sorry, for not seeing this earlier.

But this is not possible within TerrariumPI itself. I only work with max 24 hours.

But this can be done with a cron job. You can schedule a cron job https://crontab.guru/#0_20_*/2_*_* that will run a script that will toggle the relay on through the API, waits 30 sec, and toggles it off with the API.

But this means, that you cannot use that relay on an area of an enclosure. Else that will bite each other.
